### PR TITLE
MM-50238 - Add a Success Page for Changes to Annual Subscriptions.

### PR DESCRIPTION
**Pull request by ConorMacpherson**
#### Summary

When changing from a monthly-based payment interval to an annual-based, upon success, users would see the following success screen:

image

This change updates the wording on this success screen, specifically for subscription changes which involve updating the payment interval from monthly to annual.

#### Ticket Link

[MM-50238](https://mattermost.atlassian.net/browse/MM-50238)

#### Screenshots

|  before  |  after  |
|----|----|
| image | Screen Shot 2023-02-17 at 4 47 38 PM |

#### Release Note

```release-note
NONE
```

[MM-50238]: https://mattermost.atlassian.net/browse/MM-50238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/purchase_modal/process_payment_setup.tsx [hunk 1]
```diff
@@ -192,7 +193,7 @@ class ProcessPaymentSetup extends React.PureComponent<Props, State> {
         this.props.onBack();
     }
 
-    private sucessPage = () => {
+    private successPage = () => {
```
Comment (neallred) [inline components/purchase_modal/process_payment_setup.tsx:196]: :heavy_check_mark:

**Review by neallred — APPROVED** [review]
LGTM, thanks @ConorMacpherson !

**Comment by ConorMacpherson** [comment]
QA deferring this as setting up a workspace with a monthly plan for pre-merge testing has proven to be a bit finicky.

**Comment by amyblais** [comment]
@ConorMacpherson If this bug fix is for March 2nd cloud release, please help open a manual cherry-pick PR for ``cloud`` branch (automated cherry-picks are currently not working https://mattermost.atlassian.net/browse/CLD-5166).

**Comment by ConorMacpherson** [comment]
@amyblais Took a look through the feature, looks like it will be for the next release. I'll update the fix version tag in Jira. Sorry about that!

**Pull request merged**